src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/ArrayTrailingCommaCheckTest.java

```diff
@@ -45,7 +45,6 @@ public void testDefault()
             createCheckConfig(ArrayTrailingCommaCheck.class);
         final String[] expected = {
             "17: " + getCheckMessage(MSG_KEY),
-            "34: " + getCheckMessage(MSG_KEY),
             "37: " + getCheckMessage(MSG_KEY),
```

Comment:
Slightly outside the scope, but why isn't there a violation on 36 being requested? Doesn't it meet the requirements to require a trailing comma?